terraform-sentinel/courseBase.sh

```diff
@@ -0,0 +1,8 @@
+mkdir -p ~/terraform-sentinel
+git clone https://github.com/tr0njavolta/sentinel-demo-katacoda.git ~/terraform-sentinel
```

Comment:
Please put this code under the HashiCorp organization. You don't need to change it for tomorrow's practice session but it should be in a permanent place for the live event.